Makefile

```diff
@@ -73,10 +80,14 @@ $(OUT)/%.o: %.c
 	$(VECHO) "  CC\t$@\n"
 	$(Q)$(CC) -o $@ $(CFLAGS) -c -MMD -MF $@.d $<
 
-$(BIN): $(OBJS)
+$(BIN): $(OBJS) $(LIBGDBSTUB)
```

Comment:
Don't track `$(LIBGDBSTUB)` here. Check `wip/jit` branch for modular dependency.